webapp/security/models.py

```diff
@@ -107,6 +137,9 @@ class Release(Base):
     version = Column(String, unique=True)
     codename = Column(String, unique=True)
     lts = Column(Boolean)
+    esm = Column(Boolean)
```

Comment:
I believe this should remain as boolean, and when the upload script posts to the API, it should get the value of ESM date and return true or false

Reply:
How is that going to work for records already in the database? Consider this scenario:

- Security upload a CVE for the Ubuntu Bionic LTS (LTS=True, ESM=False, presumably)
- Time passes, it's now 2023. ESM now needs to be considered "True", but nothing in the database has changed...
- Time passes. It's now 2029. ESM is no longer considered "True", but nothing in the database has changed

Reply:
Note that LTS and ESM are not equivalent properties, they are not both time dependent in the same way. An LTS is always described as an LTS, even if the support has expired. As in, we still refer to "14.04 LTS" or "12.04 LTS", they are simply descriptions of the type of release. On the other hand, ESM is not a description of a type of release, it is a description of a state that a release might be in. As in, we offer extended support for all LTS releases for those who pay. So ESM is time-dependent: "14.04 LTS is in ESM until April 2022".

Reply:
Yes, but don't you need some type of operation to update an entry in the database? I suppose you need to do an `update` query to change this value anyway